conftest.py

```
@@ -260,6 +262,11 @@ def pytest_sessionfinish(session, exitstatus):
             if not session.config.getoption("skip_data_collector"):
                 run_must_gather(data_collector_path=_data_collector_path)
 
+    try:
+        maybe_teardown_awx_at_session_end(session.config)
+    except Exception as exp:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)